papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/controller/ProjectsController.java

```diff
@@ -104,6 +110,39 @@ private Project getValidProject(Long id) {
         return ResponseEntity.ok(sprints);
     }
 
+    @Override
+    public ResponseEntity<SprintSummaryDto> getSprintSummary(Long projectId, Long sprintId) {
```

Comment:
To będzie trzeba wyjąć na pewno z tego kontrolera - aktualnie mamy jedno podejście w kwestii podziału kontrolerów/serwisów u @kjarosh i inne u @Sopel97, także trzeba coś ustalić w tej kwestii. Mi się całkiem podobają te `*RestService` z brancha 55, ale obojętne, tylko żeby było jakoś tak spójnie

Reply:
No i na pewno podział na metody tutaj, bo nawet statyczna analiza krzyczy, że jest za długie

Reply:
I gdzieś można walidować, że sprint nie jest zamknięty jeszcze - w przypadku sprintów historycznych chyba powinniśmy raczej zwracać wszystkie istniejące `Availability` z tego sprintu (bez tworzenia tych pustych i bez filtrowania użytkowników `INACTIVE`), ale to jest inne issue

Reply:
u mnie juz jest jak na branchu 55, chyba